streams/src/main/java/org/apache/kafka/streams/state/internals/ThreadCache.java

```diff
@@ -218,6 +222,10 @@ public MemoryLRUCacheBytesIterator reverseAll(final String namespace) {
         return new MemoryLRUCacheBytesIterator(cache.reverseAllKeys(), cache);
     }
 
+    public MemoryLRUCacheBytesIterator prefixScan(final String namespace, final Bytes from, final Bytes to) {
```

Comment:
I think we do not need this method. We can just call `range()` in `CachingKeyValueStore`.